File: src/RestoNurseryBundle/Entity/Menu.php
<?php

namespace RestoNurseryBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Menu
 *
 * @ORM\Table(name="menu")
 * @ORM\Entity
 */
class Menu
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="date_day", type="string", length=10, nullable=false)
     */
    private $dateDay;

    /**
     * @var string
     *
     * @ORM\Column(name="plate1", type="string", length=255, nullable=false)
     */
    private $plate1;

    /**
     * @var string
     *
     * @ORM\Column(name="plate2", type="string", length=255, nullable=false)
     */
    private $plate2;

    /**
     * @var string
     *
     * @ORM\Column(name="plate3", type="string", length=255, nullable=false)
     */
    private $plate3;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getDateDay()
    {
        return $this->dateDay;
    }

    /**
     * @param string $dateDay
     */
    public function setDateDay($dateDay)
    {
        $this->dateDay = $dateDay;
    }

    /**
     * @return string
     */
    public function getPlate1()
    {
        return $this->plate1;
    }

    /**
     * @param string $plate1
     */
    public function setPlate1($plate1)
    {
        $this->plate1 = $plate1;
    }

    /**
     * @return string
     */
    public function getPlate2()
    {
        return $this->plate2;
    }

    /**
     * @param string $plate2
     */
    public function setPlate2($plate2)
    {
        $this->plate2 = $plate2;
    }

    /**
     * @return string
     */
    public function getPlate3()
    {
        return $this->plate3;
    }

    /**
     * @param string $plate3
     */
    public function setPlate3($plate3)
    {
        $this->plate3 = $plate3;
    }


}

File: Kindo Final/Kindo Final/src/Entities/Menu.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;


/**
 *
 * @author nada
 */
public class Menu {
    
    private int id;
    private String date_day;
    private String plate1;
    private String plate2;
    private String plate3;

    public Menu(int id, String date_day, String plate1, String plate2, String plate3) {
        this.id = id;
        this.date_day = date_day;
        this.plate1 = plate1;
        this.plate2 = plate2;
        this.plate3 = plate3;
    }

    public int getId() {
        return id;
    }

    public String getDate_day() {
        return date_day;
    }

    public String getPlate1() {
        return plate1;
    }

    public String getPlate2() {
        return plate2;
    }

    public String getPlate3() {
        return plate3;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setDate_day(String date_day) {
        this.date_day = date_day;
    }

    public void setPlate1(String plate1) {
        this.plate1 = plate1;
    }

    public void setPlate2(String plate2) {
        this.plate2 = plate2;
    }

    public void setPlate3(String plate3) {
        this.plate3 = plate3;
    }

    @Override
    public String toString() {
        return "Menu{" + "id=" + id + ", date_day=" + date_day + ", plate1=" + plate1 + ", plate2=" + plate2 + ", plate3=" + plate3 + '}';
    }

}
